content/200-orm/800-more/300-upgrade-guides/800-upgrade-from-prisma-1/03-upgrading-the-prisma-layer.mdx

```diff
@@ -5,7 +5,10 @@ title: 'Upgrading the Prisma ORM layer'
 dbSwitcher: ['postgresql', 'mysql']
 techMetaTitles:
   [
-    { name: 'mysql', value: 'Upgrading the Prisma ORM layer to Prisma ORM 2 | MySQL' },
+    {
+      name: 'mysql',
+      value: 'Upgrading the Prisma ORM layer to Prisma ORM 2 | MySQL',
```

Comment:
Just noticed this, unrelated to this PR:
"Prisma ORM 2" is not a great term. This is from the time when we talked about Prisma 1 and Prisma 2, later possibly about Prisma 3, 4 and 5. We should probably revisit these major version references and decide if for example this would be better described as "Prisma ORM version 2.x" or similar.